biogeochem/EDPhysiologyMod.F90

```diff
@@ -2941,6 +2958,11 @@ subroutine CWDInput( currentSite, currentPatch, litt, bc_in)
             elflux_diags%root_litter_input(pft) +  &
             (fnrt_m_turnover + store_m_turnover ) * currentCohort%n
 
+       ! send the part of the herbivory flux that doesn't go to litter to the atmosphere
+
+       site_mass%herbivory_flux_out = &
+            site_mass%herbivory_flux_out + &
+            leaf_herbivory * (1._r8 - herbivory_element_use_efficiency) * currentCohort%n
```

Comment:
Since it looks like 1._r8 - herbivory_element_use_efficiency is the portion that goes to the atmosphere, is the other portion only going to manure, food diet consumption, or both?  My bad if I'm not following